### ISSUE TITLE
ChecklessChess is broken

### ISSUE THREAD

**Issue by flab03** [thomas-maeder/popeye#247]
ChecklessChess was working fine in 4.55, but it is broken in 4.63 onwards. The following test should not return a solution.

```
begin
pieces white Ke1 Pb5 black Ka8
stip h+1.0
cond ChecklessChess
end
```

**Comment by thomas-maeder** [comment]
I couldn't agree with the title of this issue more: checkless chess is broken - 100%.

Your example is a typical example of "nonsense in, nonsense out".

Spending time for fixing it would be a waste of time.

**Comment by flab03** [comment]
Sorry, you're right, this bug doesn't seem to affect reasonable problems (I wrongly assumed it did). I'll file a different issue to explain what I'm really after.

ChecklessChess is still 1% broken because Popeye 4.81 doesn't find the solution to

```
begin
origin P1241297
stip dia 8.5
forsyth rs1qkbr1/p1ppp1pp/1p1S1s2/4K3/8/2PP1p2/PP1P1PPP/R1BQ1BSR
cond ChecklessChess
end
```

This is probably a different bug introduced later because this PG works with 4.71 while the nonsense h+1.0 doesn't.

**Comment by thomas-maeder** [comment]
Could you please post the solution that you think Popeye should find?

**Comment by thomas-maeder** [comment]
Fixed for 4.83.

Checkless Chess is still very broken, but at least Popeye's support for it is less broken now.
